### DISCUSSION TITLE
Obstacle Placement and Simulator Restrictions

### DISCUSSION THREAD

**Discussion by Sne4kers** [skhatiri/UAV-Testing-Competition#1] [Test Generation] [answered]
Hi!

After reading the rules, I have questions about the conditions applied to generators during the competition.

- Do I understand correctly, that generators submitted for competition are allowed to place the obstacles only in the area described in this quote? 

> The test generators are then expected to place obstacles in the simulation environment, inside a predefined area as shown in the below image. For simplicity, we keep this area fixed for all the case studies (-40<x<30, 10<y<40). 

- Is ros the only simulator that is going to be used in this competition? All of the example case studies use it, so is it guaranteed to be used in the evaluation process?

Thank you for your answers in advance!

**Comment by skhatiri** [comment, marked as answer]
Hi, 
Thanks for the questions. 

>Do I understand correctly, that generators submitted for competition are allowed to place the obstacles only in the area described in this quote?

- yes, all the competition case studies are designed in a way so that the mentioned area (-40<x<30, 10<y<40) is within the critical zone of the flight path. 
- This restriction is to first, avoid putting obstacles too close to the planned waypoints and making the flight not feasible, and second, to help you with reducing the search space size.

>Is ros the only simulator that is going to be used in this competition? All of the example case studies use it, so is it guaranteed to be used in the evaluation process?

- yes, the only thing that will change for the evaluation case studies is the mission plan, as already demonstrated in the [given case studies](https://github.com/skhatiri/UAV-Testing-Competition/tree/master/snippets/case_studies). The simulator and other configurations will stay the same. 
- Technically, other simulator options do not support the obstacle avoidance system. 

Of course, We are open to suggestions that improve the competition.

Good luck with the competition!
